polaris-react/build/esm/styles.css

```diff
@@ -666,7 +689,7 @@ body {
   min-height: 100%;
   margin: 0;
   padding: 0;
-  background-color: #f1f2f4;
+  background-color: rgba(241, 242, 244, 1);
```

Comment:
SASS was previously converting `rgb`/`rgba` usages into hex. Postcss does not do this, so the output is closer to the input (ie; identical).